subsys/bluetooth/host/gatt.c

```diff
@@ -5008,16 +5005,24 @@ static int db_hash_commit(void)
 {
 	int err;
 
+	k_sched_lock();
```

Comment:
I *think* this is probably a correct use of `k_sched_lock()`, but it's a pretty rarely used function so this should get some eyes from @andyross or other kernel maintainer.s